src/main/java/com/github/dockerjava/api/model/BindPropagation.java

```diff
@@ -0,0 +1,27 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+
+/**
+ * @since {@link RemoteApiVersion#VERSION_1_24}
+ */
+public enum BindPropagation {
+    @JsonProperty("rprivate")
+    R_PRIVATE,
```

Comment:
it would be impossible to get this as string, that's why other enums using field with constructor + serialiser/deserialiser. But let's try your variant.